src/app/mail/mail-detail/mail-detail.component.ts

```diff
@@ -598,6 +598,14 @@ export class MailDetailComponent implements OnInit, OnDestroy {
     this.store.dispatch(new GetMailDetail({ messageId, folder: this.mailFolder }));
   }
 
+  downloadAllAttachments(mail: Mail) {
+    if (mail && mail.attachments) {
```

Comment:
https://deepsource.io/gh/CTemplar/webclient/run/e46c9758-0d22-4ab7-bb24-39348d4ca04b/javascript/JS-D001

It should be:

`if (mail?.attachments) {`